### PR TITLE
Summary: MCU Tests:  Add two basic qadd , qlinear qdq tests

### PR DESCRIPTION
**Pull request by psiddh**
- Enable on CI for consistent signals
- Note that this is interim solution until a proper MCU testing standalone pipeline is ready

Test Plan:
- examples/arm/run_mcu_models_fvp.sh --target=cortex-m55 --models=qadd,qlinear ════════════════════════════════════════════════════════════════ 🏁 MCU MODEL VALIDATION SUMMARY - TARGET: cortex-m55 ════════════════════════════════════════════════════════════════

qadd         : ✅ Passed
qlinear      : ✅ Passed

Reviewers:

Subscribers:

Tasks:

Tags:

### Summary
[PLEASE REMOVE] See [CONTRIBUTING.md's Pull Requests](https://github.com/pytorch/executorch/blob/main/CONTRIBUTING.md#pull-requests) for ExecuTorch PR guidelines.

[PLEASE REMOVE] If this PR closes an issue, please add a `Fixes #<issue-id>` line.

[PLEASE REMOVE] If this PR introduces a fix or feature that should be the upcoming release notes, please add a "Release notes: <area>" label. For a list of available release notes labels, check out [CONTRIBUTING.md's Pull Requests](https://github.com/pytorch/executorch/blob/main/CONTRIBUTING.md#pull-requests).

### Test plan
[PLEASE REMOVE] How did you test this PR? Please write down any manual commands you used and note down tests that you have written if applicable.

### CODE REVIEW

File: examples/arm/aot_arm_compiler.py [hunk 1]
```diff
@@ -306,6 +319,7 @@ def forward(self, x: torch.Tensor, y: torch.Tensor):
     "qops": QuantOpTest,
     "softmax": SoftmaxModule,
     "MultipleOutputsModule": MultipleOutputsModule,
+    "qlinear": QuantLinearTest,
```
Comment (digantdesai) [inline examples/arm/aot_arm_compiler.py:324]: Add a TODO for more tests, link the tester RFC thing from @AdrianLundell  here?

File: examples/arm/aot_arm_compiler.py [hunk 2]
```diff
@@ -297,6 +297,19 @@ def forward(self, x: torch.Tensor, y: torch.Tensor):
     can_delegate = True
 
 
+class QuantLinearTest(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        # Define a simple linear layer
+        self.linear = torch.nn.Linear(64, 32)
```
Comment (digantdesai) [inline examples/arm/aot_arm_compiler.py]: let's use prime numbers instead of powers of two..

File: examples/arm/run_mcu_models_fvp.sh [hunk 3]
```diff
@@ -257,6 +257,7 @@ for model in "${MODELS[@]}"; do
         -m "$model" \
         --target="$ETHOS_TARGET" \
         --quantize \
+        --enable_qdq_fusion_pass \
```
Comment (digantdesai) [inline examples/arm/run_mcu_models_fvp.sh:260]: always enable?

**Review by digantdesai — APPROVED** [review]

File: examples/arm/aot_arm_compiler.py (hunk 1 above)
Comment (psiddh) [reply examples/arm/aot_arm_compiler.py:324]: done

File: examples/arm/aot_arm_compiler.py (hunk 2 above)
Comment (psiddh) [reply examples/arm/aot_arm_compiler.py]: done

File: examples/arm/run_mcu_models_fvp.sh (hunk 3 above)
Comment (psiddh) [reply examples/arm/run_mcu_models_fvp.sh:260]: Few internal CI tests were failing, I will  fix them first and then enable
